deps/v8/include/v8-version.h

```diff
@@ -11,7 +11,7 @@
 #define V8_MAJOR_VERSION 6
 #define V8_MINOR_VERSION 1
 #define V8_BUILD_NUMBER 534
-#define V8_PATCH_LEVEL 36
+#define V8_PATCH_LEVEL 38
```

Comment:
Is there another open patch going to 37?

Reply:
the change in `flag-definitions.h` is the patch37

Reply:
exactly. This patch includes both 37 + 38